tutor/specs/helpers/analytics.spec.js

```diff
@@ -23,12 +23,52 @@ describe('Analytics', function() {
   });
 
   it('skips sending pageviews if ga isn\'t present', function() {
-    Analytics.setTracker(undefined);
+    Analytics.setGa(undefined);
     Analytics.onNavigation('/course/1');
     expect(Analytics.sendPageView).not.toHaveBeenCalled()
     return undefined;
   });
 
+  it('sends the same commands to all trackers', function() {
+    // Set up a mock GA function/object
+    function Tracker(name) {
+      this.name = name;
+      this.get = function(fieldName) {
+        if ('name' == fieldName) { return this.name; }
+      };
+    }
+
+    var mockGa = function() {
+      if (arguments.length == 0) { return; }
```

Comment:
not a big deal at all, but `var` usage is frowned on with modern js.   Typically you use `const` for variables that won't be re-assiged and `let` for when they will be.

Awesome work on figuring out the stubs and putting them to good use 😁